es6-shim.js

```diff
@@ -1240,30 +1240,30 @@
 
   var MathShims = {
     acosh: function acosh(value) {
-      var x = Number(value);
-      if (Number.isNaN(x) || value < 1) { return NaN; }
-      if (x === 1) { return 0; }
-      if (x === Infinity) { return x; }
-      return Math.log(x / Math.E + Math.sqrt(x + 1) * Math.sqrt(x - 1) / Math.E) + 1;
+      value = Number(value);
```

Comment:
I just haven't yet gone through and made it consistent :-) `var x = Number(value)` seems to be a good standard for all the Math and Number functions.